### ISSUE TITLE
Build works fine, but get error after bunding

### ISSUE THREAD

**Issue by SamarRizvi** [matoilic/generator-angular-lazy#31]
@matoilic @bdurrer When simply doing the build, everything works great. But after bundling, when I test, I get Unknown Provider aProvider error, and it also gets into $digest infinite loop. What kind of provider can be missing after bundling which works absolutely perfect with simple build.

**Comment by bdurrer** [comment]
You minified your code but did not define the injections explicitly (or use an gulp task which does it for you).

That's an general issue with angular and minification and there are tons of examples and helpers for it.
Please read [Implicit Annotation in the AngularJS docs](https://docs.angularjs.org/guide/di#implicit-annotation).

**Comment by SamarRizvi** [comment]
Thanks @matoilic . Since angular-lazy uses ng-strict-di with ES6 and I am using the same gulp task provided with angular-lazy, there is very little scope of an injection issue. The only place where this issue can mostly occur in such scenarios is with resolve. I injected the dependencies correctly into resolve and viola, bundle worked great. Initially, For any one facing the same issue here's an example:

```
resolve{
    checkAuth(x1,x2) {
    }
}
```

I changed it to:

```
resolve {
    checkAuth: ['x1', 'x2', (x1,x2) => {
    }]
}

```

And there were no issue after bunding.
